rancher 2.10.2

brett060102 opened the pull request:
update deployer for rancher 2.10.2

brett060102 committed:
rancher 2.10.2

brett060102 commented on data/overlayfiles/rancher-deployer/cnab/app/charts/rancher/templates/post-delete-hook-job.yaml:
```diff
@@ -20,7 +20,7 @@ spec:
       restartPolicy: OnFailure
       containers:
         - name: {{ template "rancher.name" . }}-post-delete
-          image: "{{ include "system_default_registry" . }}{{ .Values.postDelete.image.repository }}{{- if .Values.postDelete.image.digest }}@{{ .Values.postDelete.image.digest }}{{- else }}:{{ .Values.postDelete.image.tag }}{{- end }}"
```
I think that is coming in from the updated rancher charts.

brett060102 replied:
I have restored that

mbelur commented on data/overlayfiles/rancher-deployer/cnab/app/charts/rancher/values.yaml:
```diff
@@ -106,15 +106,10 @@ privateCA: false
 
 # Override rancher image location for Air Gap installs
-rancherImage: registry.rancher.com/rancher/rancher
+rancherImage: rancher/rancher
```
shouldn't the registry still be registry.rancher.com?

brett060102 replied:
restored

brett060102 committed:
rancher 2.10.2 fix ups

Restore changes dropped when pulling code from rancher charts
in: rancher/values.yaml
   rancher image should be: registry.rancher.com/rancher/rancher for prime

in: templates/post-delete-hook-job.yaml
    need .image.digest code for azure

in: rancher/README.md
    rancherImage ref should be registry.rancher.com/rancher/rancher

guangyee reviewed (approved):
looks good to me!

mbelur reviewed (approved):
LGTM
